src/Traits/HasRoles.php

```diff
@@ -150,6 +150,8 @@ public function removeRole($role)
 
         $this->load('roles');
 
+        $this->forgetCachedPermissions();
```

Comment:
Would it make sense for the `forget` go before the re-`load`?

Reply:
I don't think it particularly matters? The load call doesn't hit the permissions cache, so far as I can tell. I notice that assignRole also leaves the cache forget until last, if that's any sort of guide.